etc/tendrl_definitions_node_agent.yaml

```diff
@@ -0,0 +1,92 @@
+---
+tendrl_schema_version: 0.1
+
+valid_objects:
+- tendrl_context
+- cpu
+- network
+- disk
+- memory
+- lvm
+- iptable
+- user
+- group
+- node
+- process
+- proc
+
+object_details:
+  cpu:
+    enabled: True
+    value: "/nodes/$node.id/cpu"
+    attrs:
+      user:
+        type: String
+        help: "Amount of time the CPU was busy executing code in user space"
+      system:
+        type: String
+      wait:
+        type: String
+      nice:
+        type: String
+      swap:
+        type: String
+      interrupt:
+        type: String
+      softirq:
+        type: String
+      steal:
+        type: String
+      idle:
+        type: String
+      active:
+        type: String
+    atoms:
+        set:
```

Comment:
we dont expose getters via flows/atoms, they are directly fetched from "value" of the object